grails-app/views/stockMovement/_summary.gsp

```diff
@@ -135,7 +135,12 @@
             </td>
             <td class="center" width="1%">
                 <div class="tag tag-alert">
-                    <format:metadata obj="${stockMovement?.shipment?.status?.code?.displayStatus?:stockMovement?.requisition?.status?.displayStatus }"/>
```

Comment:
I think it should be 
`<format:metadata obj="${stockMovement?.shipment?.status?.code?.displayStatus?:stockMovement?.statusCode}"/>`